back-end/src/main/java/com/skynet/Skymed/model/Pessoa.java

```diff
@@ -1,86 +1,118 @@
 package com.skynet.Skymed.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
 import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
 
 @Entity
 public class Pessoa {
 
 	@Id
+	@Column(name = "pes_iden")
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	private Integer pes_id;
-	private String pes_nome;
-	private String pes_cpg_cnpj;
-	private String pes_rg;
-	private String pes_telefone;
-	private String pes_email;
-	private String pes_senha;
-	
-	@ManyToOne
-	@JoinColumn(name="pes_end_id")
-	private Endereco pes_end_id;
-	private boolean pes_tipo_cliente;
-	public Integer getPes_id() {
-		return pes_id;
-	}
-	public void setPes_id(Integer pes_id) {
-		this.pes_id = pes_id;
+	private Long id;
+	@Column(name = "pes_nome")
+	private String nome;
+	@Column(name = "pes_cpf_cnpj")
+	private String cpf_cnpj;
+	@Column(name = "pes_rg")
+	private String rg;
+	@Column(name = "pes_telefone")
+	private String telefone;
+	@Column(name = "pes_email")
+	private String email;
+	@Column(name = "pes_senha")
+	private String senha;
+	@Column(name = "pes_tipo_cliente")
+	private boolean tipo_cliente;
+	@OneToOne
+	@JoinColumn(name = "pes_end_iden")
+	private Endereco endereco;
+
+	public Pessoa() {
 	}
-	public String getPes_nome() {
-		return pes_nome;
+
+	public Long getId() {
+		return id;
 	}
-	public void setPes_nome(String pes_nome) {
-		this.pes_nome = pes_nome;
+
+	public void setId(Long id) {
+		this.id = id;
 	}
-	public String getPes_cpg_cnpj() {
-		return pes_cpg_cnpj;
+
+	public String getNome() {
+		return nome;
 	}
-	public void setPes_cpg_cnpj(String pes_cpg_cnpj) {
-		this.pes_cpg_cnpj = pes_cpg_cnpj;
+
+	public void setNome(String nome) {
+		this.nome = nome;
 	}
-	public String getPes_rg() {
-		return pes_rg;
+
+	public String getCpg_cnpj() {
+		return cpf_cnpj;
 	}
-	public void setPes_rg(String pes_rg) {
-		this.pes_rg = pes_rg;
+
+	public void setCpg_cnpj(String cpg_cnpj) {
```

Comment:
```suggestion
	public void setCpg_cnpj(String cpg_cnpj) {
```
cpg né, achou q eu nao ia ver